[T2] `bgp/test_bgp_session_flap.py` fails with `KeyError`

### Issue Description

We see this issue on Arista T2 modulars when there is a single-asic LC present
```
    @pytest.fixture(scope='module')
    def setup(tbinfo, nbrhosts, duthosts, enum_frontend_dut_hostname, enum_rand_one_frontend_asic_index):
        duthost = duthosts[enum_frontend_dut_hostname]
        asic_index = enum_rand_one_frontend_asic_index
        namespace = duthost.get_namespace_from_asic_id(asic_index)

        bgp_facts = duthost.bgp_facts(instance_id=asic_index)['ansible_facts']
        neigh_keys = []
        tor_neighbors = dict()
        neigh_asn = dict()
        for k, v in bgp_facts['bgp_neighbors'].items():
            if 'asic' not in v['description'].lower():
                neigh_keys.append(v['description'])
                neigh_asn[v['description']] = v['remote AS']
>               tor_neighbors[v['description']] = nbrhosts[v['description']]["host"]
E               KeyError: 'cmp314-4'
```

The problem is it's checking for the iBGP neighbor of the remote single-asic LC against the `nbrhosts` which holds the list of eBGP neighbors:
```
(Pdb) p nbrhosts.keys()
dict_keys(['ARISTA04T1', 'ARISTA04T3', 'ARISTA01T1', 'ARISTA06T1', 'ARISTA06T3', 'ARISTA01T3', 'ARISTA03T3', 'ARISTA03T1'])
```

### Results you see

`bgp/test_bgp_session_flap.py` fails with `KeyError` when run on a T2 with a single-asic LC

### Results you expected to see

`bgp/test_bgp_session_flap.py` should pass, likely the iBGP neighbor should be ignored/skipped

### Is it platform specific

generic

### Relevant log output

_No response_

### Output of `show version`

_No response_

### Attach files (if any)

_No response_

--- a/tests/bgp/test_bgp_session_flap.py
+++ b/tests/bgp/test_bgp_session_flap.py
@@ -66,7 +66,8 @@ def setup(tbinfo, nbrhosts, duthosts, enum_frontend_dut_hostname, enum_rand_one_
     tor_neighbors = dict()
     neigh_asn = dict()
     for k, v in bgp_facts['bgp_neighbors'].items():
-        if 'asic' not in v['description'].lower():
+        # Skip iBGP neighbors
+        if "INTERNAL" not in v["peer group"] and "VOQ_CHASSIS" not in v["peer group"]:
             neigh_keys.append(v['description'])
             neigh_asn[v['description']] = v['remote AS']
             tor_neighbors[v['description']] = nbrhosts[v['description']]["host"]

